governance.md

```diff
@@ -0,0 +1,55 @@
+
+## The Executive Directors
+
+The executive directors have two sets of responsibilities:
+
+1. Conduct the mundane business associated with the foundation.
+2. Ratify _Working Group Approval_ documents, making them Approved Drafts, which are then published as complete artifacts.
+
+The current executive directors of the CNAB Foundation are (in alphabetical order):
+
+- Matt Butcher (Microsoft)
+- Chris Crone (Docker)
+
+Provisionally, new executive directors are nominated by existing executive directors by unanimous vote. In the event that executive directors cannot name new directors, the working groups may nominate and then vote on new executive directors, with a 2/3 supermajority required for successful nomination. No one company may hold 50% or more of the voting power of the executive directorship.
+
+## The Working Group Lifecycle
+
+This process governs the motion of a specification from rough draft through full standardization. While it does not necessarily follow the typical software lifecycle, we have approximately adapted it for reference implementations as well. A document is termed a _deliverable_ as it passes through various phases from _pre-draft deliverable_ to _accepted deliverable_, at which point it becomes a final specification.
+
+### Versioning
+
+Specifications shall each track their own versions. A specification will call out a target version (such as 1.0), and then also include its phase (such as Pre-Draft, see below) to indicate progress towards the target version.
+
+For example, one of the specifications in this repository is the CNAB Core specification. The CNAB Core specification should be formally referenced as _Cloud Native Application Bundle Core 1.0.0_. The phase of the process MAY be appended as a stability marker: _Cloud Native Application Bundle Core 1.0.0-Draft_. It MAY be abbreviated to _CNAB1_ or _CNAB1-Draft_.
+
+Other specifications must follow the same nomenclature.
+
+### Development Process
+
+The specification will proceed through the following phases:
+
+- *Pre-Draft (PD):* Any working group participant or contributor may submit a proposed initial draft document as a candidate for this working group. The Working Group Chair may designate such a submission as a Pre-Draft Document.
+- *Draft (Draft):* A Pre-Draft may be approved by the working group, in which case it becomes an official draft under the auspices of the CNAB specification working group. The working group will continue to revise the draft until it is in a state the group sees as fit for standardization.
+- *Working Group Approval (GA):* When the working group believes a draft fit for standardization, the group formally approves the draft and submits it for final approval.
+- *Final Approval (AD):* The Executive Director (or a named proxy) may grant Final Approval to a draft with Working Group Approval. At this point, the work is now designated an Approved Deliverable and is no longer a draft.
```

Comment:
Since there are two (or more) of us, it would be good to clarify that we need a majority (not sure if this is simple, super, or unanimous)